fxa-client/ffi/src/lib.rs

```diff
@@ -22,13 +22,21 @@ use fxa_client::ffi::*;
 
 /// Convenience function over [fxa_get_custom_config] that provides a pointer to a [Config] that
 /// points to the production FxA servers.
+///
+/// # Safety
+///
+/// Please note that most methods taking a [Config] as argument will take ownership of it and
+/// therefore the callers shall **not** free the [Config] afterwards.
+///
+/// A destructor [fxa_config_free] is provided for releasing the memory for this
+/// pointer type.
 #[no_mangle]
-pub extern "C" fn fxa_get_release_config(err: &mut ExternError) -> *mut Config {
-    call_with_result(err, Config::release)
+pub unsafe extern "C" fn fxa_get_release_config(content_base: *const c_char, client_id: *const c_char, err: &mut ExternError) -> *mut Config {
+    call_with_result(err, Config::release(rust_str_from_c(client_id),rust_str_from_c(redirect_uri)))
```

Comment:
I think you forgot `redirect_uri` as a param here.
Small nit: spaces after commas.